package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "vite-project",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.1.0",
   "type": "module",
   "scripts": {
+    "start": "npm run dev",
```

Comment:
`npm start` is a default script that should be supported. https://docs.npmjs.com/cli/v10/commands/npm-start